src/Block/GalleryBlockService.php

```diff
@@ -96,7 +96,8 @@ public function configureSettings(OptionsResolver $resolver): void
             'format' => false,
             'pauseTime' => 3000,
             'startPaused' => false,
-            'template' => 'SonataMediaBundle:Block:block_gallery.html.twig',
+            'wrap' => true,
```

Comment:
This is missing on master since last CI merge, not sure why.

Reply:
Does it have to do with ca9a642 ?

Reply:
Not sure, as it present on current `3.x` https://github.com/sonata-project/SonataMediaBundle/blob/3.x/src/Block/GalleryBlockService.php#L97 also https://github.com/sonata-project/SonataMediaBundle/blob/bb60020eafa6874f214b1b129437db4e8ac89afb/src/Block/GalleryBlockService.php#L97 commit on master 18 days ago, and your commit is from 2015.

Reply:
I think CI fails because of this.

Reply:
be careful because it probably was removed on master directly. I remember this wrap attribute. can you check master?

Reply:
It was indeed purposefully removed directly on master: https://github.com/sonata-project/SonataMediaBundle/pull/890

Reply:
ah, sorry, will remove it.